accounts/keystore/keystore.go

```diff
@@ -539,10 +539,12 @@ func (ks *KeyStore) ImportECDSA(priv *ecdsa.PrivateKey, passphrase string) (acco
 }
 
 func (ks *KeyStore) importKey(key Key, passphrase string) (accounts.Account, error) {
-	a := accounts.Account{Address: key.GetAddress(), URL: accounts.URL{Scheme: KeyStoreScheme, Path: ks.storage.JoinPath(keyFileName(key.GetAddress()))}}
+	accDir := ks.storage.JoinPath(keyFileName(key.GetAddress()))
+	a := accounts.Account{Address: key.GetAddress(), URL: accounts.URL{Scheme: KeyStoreScheme, Path: accDir}}
 	if err := ks.storage.StoreKey(a.URL.Path, key, passphrase); err != nil {
 		return accounts.Account{}, err
 	}
+	fmt.Println("Your account is imported at", accDir)
```

Comment:
```suggestion
	logger.Info("Your account is imported at", "path",accDir)
```
We rarely use `fmt.println` for giving information. How about using `logger.Info`?

Reply:
Thanks for the suggestion. When the `account import` command is used, all prints are output to `stdout` using `fmt.Println`. That's why I just followed their usage. Of course, we can push it to logs, but it may be harder or worse to find that.

Reply:
`personal` API also calls this method, and this stdout output can make the node log massy. If you are intended to print this message with homi or node commands, you better print this in `homi` and `nodecmd` packages.

Reply:
Hi @aidan-kwon, Thanks for the advice. I found the function `accountImport()` is only called when the import command is triggered. I think placing this print log at the end of that function seems to be the correct location.